pkgs/applications/audio/sptlrx/default.nix

```diff
@@ -2,25 +2,23 @@
 
 buildGoModule rec {
   pname = "sptlrx";
-  version = "0.2.0";
+  version = "1.0.0";
 
   src = fetchFromGitHub {
     owner = "raitonoberu";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-b38DACSdnjwPsLMrkt0Ubpqpn/4SDAgrdSlp9iAcxfE=";
+    sha256 = "sha256-UDxmUc902A6+DC254wyvjSzNs95K7QIuDW+24o8VCCc=";
   };
 
-  vendorSha256 = "sha256-/fqWnRQBpLNoTwqrFDKqQuv1r9do1voysBhLuj223S0=";
+  vendorSha256 = "sha256-t9Mkszzuw7YtBnADsZDjwN2AA6MuQH4+zzDiHe302A4=";
 
   ldflags = [ "-s" "-w" ];
 
   passthru = {
     updateScript = nix-update-script { attrPath = pname; };
     tests.version = testers.testVersion {
       package = sptlrx;
-      # TODO Wrong version in `0.2.0`. Has been fixed upstream.
-      version = "v0.1.0";
```

Comment:
Maybe you can try `version = "v${version}";`, testers.testVersion uses `grep -Fw` to look for version string (and ofborg says passthru.tests are failing)

Reply:
Oh yeah, I didn't even notice that. Should be fixed now.